package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-authenticate-me",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
We don't normally bump versions until a release. FYI, version number bumps don't need a PR, and should just be done right before a release.